Assets/Scripts/Dungeon/ProcGen/DungeonGenerator.cs

```diff
@@ -2,13 +2,16 @@
 {
     using UnityEngine;
     using MapGen;
+    using Util;
 
     public class DungeonGenerator : MonoBehaviour
     {
         [SerializeField]
         private DungeonMap mapTemplet = null;
         [SerializeField]
         private Room roomTemplet = null;
+        [SerializeField]
+        private Enums.EnemyTypes[] procEnemies;
```

Comment:
Don't add this here, this is not the responsibility of this class.  This is already implemented in Room.cs